CMakeLists.txt.ros1

```diff
@@ -26,7 +26,8 @@ catkin_package(INCLUDE_DIRS
                sensor_msgs
                geometry_msgs
```

Comment:
`std_msgs` and `sensor_msgs` can be removed as dependencies. They were used by the predecessor project, and made it into these build files as a result.

Reply:
See https://github.com/calderpg/common_robotics_utilities/pull/6/commits/c10dc233726126147cf820a91d05f7a5c3a32ec2.